### PR TITLE
Change success message at end of training

### LINKED ISSUE #155: Change success message for end of training
Currently the training loop prints out "termintating process" in red for both success and failure, let's fix it so that success prints out happy text in green. This way users aren't worried about a failure.

### PR DESCRIPTION
**Pull request by alinaryan**
Changes the verbiage and color of the success message to green. 

Fixes #155

### CODE REVIEW

**Review by RobotSail — APPROVED** [review]
LGTM

**Review by aakankshaduggal — APPROVED** [review]

File: src/instructlab/training/main_ds.py [hunk 1]
```diff
@@ -678,7 +678,7 @@ def run_training(torch_args: TorchrunArgs, train_args: TrainingArgs) -> None:
         if "process" not in locals() or process is None:
             return
 
-        print("\033[91mTerminating process 🤖\033[0m")
```
Comment (RobotSail) [inline src/instructlab/training/main_ds.py:681]: Will this be only for successful completions or all completions? Wondering whether we should somehow check the success status here.

Comment (RobotSail) [reply src/instructlab/training/main_ds.py:681]: It looks like you should be able to achieve this by using the return value from `process.poll()`:

```py
if process.poll() == 0:
  print("success!")
else:
  print("failed :(")
```

Comment (alinaryan) [reply src/instructlab/training/main_ds.py:681]: Thanks for the suggestion, Oleg! How's this?

Comment (RobotSail) [reply src/instructlab/training/main_ds.py:681]: It's perfect, thank you!

**Review by RobotSail — APPROVED** [review]
LGTM

**Review by cdoern — COMMENTED** [review]
I am fine with this merging as long as we are 100% sure it will not introduce unexpected success/failed msgs. Is this something we want to risk for GA? I might be over-exaggerating here though since this seems pretty straight forward

**Comment by RobotSail** [comment]
@cdoern This should be fine but we can test it in our e2e

**Comment by Maxusmusti** [comment]
Pending test from Oleg, will approve/merge this PR once all looks good there. Will include this in a 0.4.0 release alongside James' PR once tested (#160)

**Comment by RobotSail** [comment]
Tested this locally and it worked flawlessly. Thank you again for this great PR @alinaryan !

**Pull request merged**